README.md

```diff
@@ -48,6 +51,50 @@ Each of the extensions / integrations has to be installed separately.
 | [`fuel-jackson`](./fuel-jackson) | _Deserialization_: [`Jackson`](https://github.com/FasterXML/jackson-module-kotlin)
 | [`fuel-moshi`](./fuel-moshi) | _Deserialization_: [`Moshi`](https://github.com/square/moshi)  |
 
+### Jitpack
+
+```kotlin
```

Comment:
[Q] Should we care about our groovy user? 😺 😈

Reply:
Groovy uses maven right? I don't mind having more instructions in here, but don't groovy users usually know how to do this :P ?

Reply:
groovy users should know how to do the same thing ..if this turns out to be a problem we can still look for how to make dual language codeblocks.. like on the gradle docs